pkgs/development/libraries/libsixel/default.nix

```diff
@@ -1,30 +1,45 @@
-{lib, stdenv, fetchFromGitHub}:
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, libbsd
+, gdk-pixbuf
+, gd
+, libjpeg
+, pkg-config
+}:
 stdenv.mkDerivation rec {
-  version = "1.8.6";
   pname = "libsixel";
+  version = "1.10.1";
 
   src = fetchFromGitHub {
+    owner = "libsixel";
     repo = "libsixel";
     rev = "v${version}";
-    owner = "saitoha";
-    sha256 = "1saxdj6sldv01g6w6yk8vr7px4bl31xca3a82j6v1j3fw5rbfphy";
+    sha256 = "sha256-ACypJTFjXSzBjo4hQzUiJOqnaRaZnYX+/NublN9sbBo=";
   };
 
-  configureFlags = [
-    "--enable-tests"
+  buildInputs = [
+    libbsd gdk-pixbuf gd
   ];
 
-  doCheck = true;
```

Comment:
Adding `-Dtests=enabled` to `mesonFlags` brings the tests back, which pass for me on nixos x86_64

Reply:
Fixed.